tests/travis/docker-run-mysql-or-mariadb.sh

```diff
@@ -40,3 +40,11 @@ while true; do
       ;;
   esac
 done
+
+if [[ "$TLS" == "yes" ]]
```

Comment:
I noticed this in several of our packages, is that a standard way of doing this? I mean we could use set `TLS` to `true` which is a program that always succeeds, and then write:
```suggestion
if ${TLS:false}
```

Reply:
Honestly, I haven't ever seen the syntax like `if ${TLS:false}` being used used but have seen `if [[ "$TLS" == "yes" ]]` or even `if [[ "x$TLS" == "xyes" ]]`. AFAIK, it's all about compatibility with other shells, etc.

I don't think any of us wants to spend our time on coming up with a Bash coding standard right now :-)